infra/terraform/modules/network/proxy_public.tf

```diff
@@ -0,0 +1,61 @@
+################### Public A ######################
```

Comment:
This is also about subnet, and the backend API may be located here non ?

Reply:
Currently, the public_subnet exists to keep the internet facing loadbalancers. I think that I'll keep that name for the moment and I'll made the refacto when Ill work on api.